src/Compilers/CSharp/Portable/DocumentationComments/DocumentationCommentIDVisitor.PartVisitor.cs

```diff
@@ -187,7 +187,7 @@ public override object VisitNamedType(NamedTypeSymbol symbol, StringBuilder buil
                 {
                     // Special case: dev11 treats types instances of the declaring type in the parameter list
                     // (and return type, for conversions) as constructed with its own type parameters.
-                    if (!_inParameterOrReturnType && TypeSymbol.Equals(symbol, symbol.ConstructedFrom, TypeCompareKind.ConsiderEverything2))
+                    if (!_inParameterOrReturnType && TypeSymbol.Equals(symbol, symbol.ConstructedFrom, TypeCompareKind.AllNullableIgnoreOptions))
```

Comment:
>AllNullableIgnoreOptions [](http://example.com/codeflow?start=119&length=24)

Can we use ```AllIgnoreOptions``` here?

Reply:
@AlekseyTs Do you have a scenario in mind where it matters?

Reply:
> Do you have a scenario in mind where it matters?

See my other comment. Usually, when we are replacing ```ConsiderEverything2```, we should demonstrate that options that we are not ignoring matter, Not the other way around.